common/big.go

```diff
@@ -94,6 +97,45 @@ func LogBig(diff *big.Int) *big.Int {
 	return bigBits
 }
```

Comment:
There appear to be multiple instances of LogBig that are used, as well as multiple definitions of MantBits.

Reply:
removed, the multiple instances of both mantbits and logbig